src/phases/query/Multiplanner.yml

```diff
@@ -147,3 +147,95 @@ Hide14IndexesTemplate:
         - {OperationName: RunCommand, OperationCommand: {collMod: *coll2, index: {hidden: true, name: index14}}}
         - {OperationName: RunCommand, OperationCommand: {collMod: *coll2, index: {hidden: true, name: index15}}}
         - {OperationName: RunCommand, OperationCommand: {collMod: *coll2, index: {hidden: true, name: index16}}}
+
+fraction: &frac {^NumExpr: {withExpression: "resultCount / docCount", andValues: {
+                      resultCount: &resultCount {^Parameter: {Name: "resultCount", Default: {unused: "Invalid result count."}}},
+                      docCount: {^Parameter: {Name: "docCount", Default: {unused: "Invalid docCount."}}}}}}
+
+FindWithNExpectedResults:
+  Name: FindWithNExpectedResults
+  Type: RunCommand
```

Comment:
As I mentioned on slack, the fundamental problem with this is that we don't exhaust or close the cursor, returned by the find command here.

This leads to two problems:
1. We don't test actual whole query execution.
2. We leave open cursors that may lead to RAM overflow.

We can solve it by setting batchSize to be bigger than the number of expected results.
However, if result set is bigger than 16 MBs, it will lead to the same problems.

The biggest issue with this is that the problem is "silent" - we won't notice it and just get bad data.

So I suggest that you file a ticket to support let option in CrudActor and do it.
It will be useful addition to genny and you get to write some C++ code instead of genny yaml files!

We can do this workload later after you are done with this.

If we want to start gathering perf data for this,
let's add cursor batchSize option to this query, so we can start getting data while figuring out let in CrudActors.